src/test/java/codelook/jpa/BookInfoTest.java

```diff
@@ -0,0 +1,40 @@
+package codelook.jpa;
+
+
+import codelook.jpa.objects.AuthorInfo;
+import codelook.jpa.objects.BookInfo;
+import org.junit.Before;
+import org.junit.Test;
+
+
+import java.util.List;
+
+import static org.junit.Assert.*;
+public class BookInfoTest {
```

Comment:
Related to earlier comment about REST controller - could make tests more representative by calling the REST methods instead of querying the database directly